.storybook/pages/CoursePlannerStep1/CoursePlannerStep1.tsx

```diff
@@ -18,84 +20,96 @@ import {
 
 import '../../../src/components/Utilities/Spacing.css';
 import '../../../src/components/Utilities/TypographyUsage.css';
-
 import CompassCenter from '../../static/compass-center.svg';
 
-export const CoursePlannerStep1 = () => (
-  <>
-    <Panel className="!mb-8" variant="squared">
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+}
+
+export const CoursePlannerStep1 = (className: Props) => {
```

Comment:
This is another "page"; we don't need the `className` prop.
```suggestion
export const CoursePlannerStep1 = () => {
```